pkg/ext-proc/backend/provider.go

```diff
@@ -89,6 +90,15 @@ func (p *Provider) Init(refreshPodsInterval, refreshMetricsInterval time.Duratio
 		}
 	}()
 
+	// Periodically flush prometheus metrics for inference pool
+	flushPrometheusMetricsInterval := 10 * time.Second
```

Comment:
I think we may want to do this more frequently sine we could use those metrics for HAP, we probably need it as a flag.

Reply:
Make it a flag.